src/CentrifugeRouter.sol

```diff
@@ -169,6 +191,52 @@ contract CentrifugeRouter is Auth, ICentrifugeRouter {
         }
     }
 
+    /// @inheritdoc ICentrifugeRouter
+    function cancelRedeemRequest(address vault, address controller) external payable protected {
+        validateController(vault, controller);
+        IERC7540Vault(vault).cancelRedeemRequest(0, controller);
+    }
+
+    /// @inheritdoc ICentrifugeRouter
+    function claimCancelRedeemRequest(address vault, address receiver, address controller) external payable protected {
+        validateController(vault, controller);
+        IERC7540Vault(vault).claimCancelRedeemRequest(0, receiver, controller);
+    }
+
+    // --- Transfer ---
+    /// @inheritdoc ICentrifugeRouter
+    function transfer(address asset, bytes32 recipient, uint128 amount) external protected {
```

Comment:
it will be user friendlier if it accepts `address` and just `toBytes32()` when you call `poolManager.transfer`

Reply:
@peculiarity @AStox this doesn't work because it's transferring to Centrifuge Chain, where addresses are 32 bytes